button.py

```diff
@@ -0,0 +1,29 @@
+class Button:
+    def __init__(self, image, pos, text_input, font, base_color, hovering_color):
```

Comment:
Only if you have the time, can you use type hints what each parameter does, if others want to improve the code later on!

For the rest, the button class looks very clean!